katas/content/superposition/three_states_two_qubits_phases/Placeholder.qs

```diff
@@ -0,0 +1,17 @@
+namespace Kata {
+    open Microsoft.Quantum.Math;
+
+    operation ThreeStates_TwoQubits_Phases (qs : Qubit[]) : Unit {
+        // Implement your solution here...
+
+    }
+
+    // You might find this helper operation from an earlier task useful.
+    operation ThreeStates_TwoQubits (qs : Qubit[]) : Unit is Adj {
+
+        let theta = ArcSin(1.0 / Sqrt(3.0));
+        Ry(2.0 * theta, qs[0]);
+
```

Comment:
There is not a lot of space in the coding window if you work through the kata on a laptop, so let's keep helper code brief
```suggestion
```